cypress/e2e/smoke/dbot/SearchBlockMenu.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('QATEST-4249: Bot Builder - Search block menu', () => {
+  beforeEach(() => {
+    cy.c_createRealAccount()
+    cy.c_login()
+  })
+  it('Should login from app.deriv.com, opening DBot and checking of valid/invalid requests in Search bar', () => {
+    cy.c_visitResponsive('/bot#bot_builder', 'large')
+    cy.c_skipTour()
+    cy.findByPlaceholderText('Search')
+      .type('Trading')
+      .should('have.value', 'Trading')
+    cy.contains('Results for "Trading"')
```

Comment:
can you use testing library locator here?
```suggestion
    cy.findByText('Results for "Trading"')
```

Reply:
done